crates/starknet-devnet-core/Cargo.toml

```diff
@@ -36,6 +39,7 @@ openssl = { workspace = true }
 
 [dev-dependencies]
 hex = { workspace = true }
+starknet-types = { workspace = true, features = ["testing"] }
```

Comment:
There are other dependencies whose `testing` features we import, but which are listed under `[dependencies]`. Is that ok? I.e. are these testing features really used in production, and not in tests only?